proxy/src/http/sql_over_http.rs

```diff
@@ -278,13 +285,20 @@ pub async fn handle(
 
 async fn query_to_json<T: GenericClient>(
     client: &T,
-    data: QueryData,
+    data: &QueryData,
     raw_output: bool,
     array_mode: bool,
 ) -> anyhow::Result<Value> {
-    let query_params = json_to_pg_text(data.params)?;
+    let query_params = json_to_pg_text(&data.params)?;
     let row_stream = client
-        .query_raw_txt::<String, _>(data.query, query_params)
+        // TODO: query_raw_txt should be able to accept &str and Vec<String>
```

Comment:
done already in my rust-postgres PR

Reply:
Is https://github.com/neondatabase/rust-postgres/pull/21 ready for merge? I can get it merged and also bump version in this PR

Reply:
I'm happy with it. Go ahead

Reply:
fixed in https://github.com/neondatabase/neon/pull/4958/commits/776cc063c56ad1f10433805a0aa93ac2117d663f

Reply:
```suggestion
```